gemfiles/rails_4_devise_3.gemfile

```diff
@@ -7,4 +7,4 @@ gem "actionpack", "~> 4.0"
 gem "activerecord", "~> 4.0"
 gem "devise", "~> 3.2"
 
-gemspec :path => "../"
```

Comment:
If this was done by Appraisal, all good. :+1: If you edited them manually, let not do it. These are generated files, and it is better to not ever edit them manually. :slightly_smiling_face: